pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -229,6 +229,11 @@ type PoolSpec struct {
 	ErasureCoded ErasureCodedSpec `json:"erasureCoded"`
 }
 
+type PoolStatus struct {
```

Comment:
Could we define a single `Status` struct? Seems like the status structs for the CRDs will all be the same with the phase and message. We can always create more types in the future if we need to differentiate.

Reply:
yup

Reply:
updated with a function